lab-yunjoo/lib/note-crud.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const Note = require('../model/note');
+const AppErr = require('../lib/app-error');
+const debug = require('debug')('rito:note-crud');
+
+exports.createNote = function(reqBody){
```

Comment:
Does it really make sense to be calling this a reqBody here? The whole point of controllers is that we aren't forcing crud operations to only happen through server routes, but calling it reqBody here doesn't make that clear.